sycl/test/kernel-and-program/kernel-and-program.cpp

```diff
@@ -52,18 +52,16 @@ int main() {
     assert(data == 1);
 
     // OpenCL interoperability kernel invocation
-    // TODO add set_args(cl::sycl::sampler) use case once it's supported
     if (!q.is_host()) {
       cl_int err;
```

Comment:
I guess `err` should me moved inside the next scope. I don't see it used outside of this scope.

Reply:
Moved.